test/language/statements/for-in/head-const-bound-names-let.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) 2016 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: The declaration may not contain a binding for `let`
+negative: SyntaxError
+info: |
+    It is a Syntax Error if the BoundNames of ForDeclaration contains "let".
+id: sec-for-in-and-for-of-statements
+es6id: 13.7
+flags: [noStrict]
+---*/
+
+for (const let in {}) {}
```

Comment:
It is an early error, but due to another part of the specification (strict mode
restrictions on identifiers). This test declares `noStrict` in order to keep
the semantics under test limited to the IterationStatement's early errors (we
maintain explicit tests for strict mode identifier restrictions elsewhere).